docs/developers-guide/driver-changelog.md

```diff
@@ -18,52 +18,41 @@ title: Driver interface changelog
-
-  - `metabase.driver.sql.query-processor/format-honeysql` is now a multimethod, mainly so you can bind
-    `*compile-with-inline-parameters*` if you need to always compile without parameterization.
-
-  - The dynamic variable `metabase.driver/*compile-with-inline-parameters*` (default `false`) has been added; drivers
-    that can generate parameterized queries should look at its value in their implementation of
-    `metabase.driver/mbql->native` and adjust their output accordingly. For `:sql-jdbc`-based drivers that support
-    parameterization, this is handled in the shared `metabase.driver.sql.query-processor` code, so you shouldn't need
-    to adjust anything here. For `:sql` drivers that do not support JDBC-style parameterized queries you can implement
-    `format-honeysql` and bind `*compile-with-inline-parameters*` as discussed above. See the `:athena` driver for an
-    example of how to do this.
-
-  - `metabase.driver.sql.util.unprepare/unprepare`, which took a parameterized SQL string and de-parameterized or
-    "unprepared" it, has been removed. Instead, if you need a query with parameters spliced directly into the SQL,
-    bind `metabase.driver/*compile-with-inline-parameters*` as discussed above.
-
-  - Similarly, the driver method `metabase.driver/splice-parameters-into-native-query` has been marked deprecated, and
-    the default implementation will throw an Exception if called. Rework code that generates parameterized queries and
-    then calls `unprepare` or `splice-parameters-into-native-query` with code that generates queries with inlined
-    parameters in the first place as discussed above. Tests can use
-    `metabase.query-processor.compile/compile-with-inline-parameters` if needed.
-
-   - `metabase.query-processor.compile/compile-and-splice-parameters` has been removed; replace usages with
-     `metabase.query-processor.compile/compile-with-inline-parameters`.
-
-  - The three-arity of `metabase.driver.sql.query-processor/format-honeysql` (which had an additional parameter for
-    Honey SQL version) has been removed; replace all usages with the two-arity version. Honey SQL 2 has been the only
-    supported version since Metabase 0.49.0.
-
-## Metabase 0.50.16
-
-  - `:type/fingerprinting-unsupported` has been added in the `metabase.types` namespace. Similar to
-    `:type/field-values-unsupported` for field values scanning, it is used to determine whether a specific field
-    should have its fingerprint computed or not. At the time of writing that logic is performed in
-    `metabase.sync.analyze.fingerprint/fields-to-fingerprint-base-clause`.
-
-  - `:type/Large` has been also been added in the `metabase.types` namespace. It can be used by driver authors to
-    signal that a specific field contains large enough values to skip fingerprinting or field values scanning. It
-    can be used for other purposes as well in the future. Examples include Oracle CLOB or Postgres JSON columns.
```

Comment:
These were meant to be after the entries for 51.

See https://github.com/metabase/metabase/pull/44654/files#diff-36f2d80db96f8bcb0e52d18bc8bd958041f6dbde505a35d9bc985ce16be16c06R57-R66